### ISSUE TITLE
Inline style in host element of the component

### ISSUE THREAD

**Issue by Hulva** [start-javascript/sb-admin-material#4]
![untitled](https://user-images.githubusercontent.com/6812031/41649305-e572aafa-74ad-11e8-8fe9-94cb6a20554a.png)

![image](https://user-images.githubusercontent.com/6812031/41649382-066f9114-74ae-11e8-9a9b-14f5b5a55b2a.png)

I've cloned the mater code, and run it, turns out like this. I found it has added inline styles in host element of the component. Did anyone has a clue?

**Comment by lwazevedo** [comment]
The solution I found is to remove fxFlex from the parent tags, and left it only on daughters.

In the following code, what was in bold was what I removed. I did this for all the components that are in trouble.

From what I noticed the newer version of flexlayout does not have this option in parent tags.

![screenshot_2](https://user-images.githubusercontent.com/17619473/41671056-642bfd32-748c-11e8-810f-d88a53b1af11.png)


Tanks
